pi5/pinctrl_utils.go

```diff
@@ -238,6 +222,25 @@ func setGPIONodePhysAddrHelper(currNodePath string, physAddress uint64, numCAddr
 	return setGPIONodePhysAddrHelper(currNodePath, physAddress, numCAddrCells)
 }
 
+// Uses Information Stored within the 'reg' property of the child node and 'ranges' property of its parents to map the child's physical address into the dev/gpiomem space
+func (b *pinctrlpi5) setGPIONodePhysAddr(nodePath string) error {
+
+	var err error
+	currNodePath := dtBaseNodePath + nodePath // initially: /proc/device-tree/axi/pcie@120000/rp1/gpio@d0000
+	invalidAddr := uint64(math.NaN())
+	numCAddrCells := uint32(0)
+
+	/* Call Recursive Function to Calculate Phys Addr. Works way up the Device Tree, using the information
```

Comment:
Why Is Every Word Capitalized?

Reply:
no reason, i will fix that

Reply:
I see this is resolved but it is still capitalized - reminder to change

Reply:
thanks for catching that - there was another place that had the same issue. if you pull both should be resolved